apps/documentation/src/app/examples/radio/radio.tailwind.example.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, signal } from '@angular/core';
+import { NgpRadioGroup, NgpRadioIndicator, NgpRadioItem } from 'ng-primitives/radio';
+
+@Component({
+  selector: 'app-radio-tailwind',
+  imports: [NgpRadioGroup, NgpRadioItem, NgpRadioIndicator],
+  template: `
+    <div
+      class="flex flex-col gap-4"
+      [(ngpRadioGroupValue)]="plan"
+      ngpRadioGroup
+      ngpRadioGroupOrientation="vertical"
+    >
+      <div
+        class="grid h-[66px] cursor-pointer grid-cols-[auto,1fr] grid-rows-2 gap-x-[0.625rem] gap-y-[0.125rem] rounded-lg bg-gray-50 px-4 py-3 text-black shadow outline-none transition-colors duration-200 data-[checked]:bg-black data-[hover]:bg-gray-100 data-[press]:bg-gray-200 data-[checked]:text-white data-[focus-visible]:outline-2 data-[focus-visible]:outline-offset-2 data-[focus-visible]:outline-blue-500 dark:bg-black dark:text-white dark:data-[checked]:bg-white dark:data-[hover]:bg-zinc-800 dark:data-[press]:bg-zinc-700 dark:data-[checked]:text-black"
```

Comment:
Just noticed a slight difference, the css example items have a white background, and a subtle border - I believe this could be achieved by adding a `ring-black/10` or similar.

<img width="347" height="261" alt="image" src="https://github.com/user-attachments/assets/7163f9a4-ed5c-48b0-b710-133f4df8b885" />

<img width="332" height="259" alt="image" src="https://github.com/user-attachments/assets/6e701365-a9dc-415f-9f4d-ddd684887c8d" />


Apart from that look great! Thank you!